src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -51,10 +51,6 @@
 
     public static final Pattern FILE_ENDING_PATTERN = Pattern.compile(FILE_ENDING_REGEX);
 
-    public static final String TUM_USERNAME_REGEX = "^([a-z]{2}\\d{2}[a-z]{3})";
-
-    public static final Pattern TUM_USERNAME_PATTERN = Pattern.compile(TUM_USERNAME_REGEX);
-
     public static final Pattern TITLE_NAME_PATTERN = Pattern.compile("^[a-zA-Z0-9_\\-\\s]*");
 
     public static final String TUM_LDAP_MATRIKEL_NUMBER = "imMatrikelNr";
```

Comment:
As this is also TUM specific (at least I guess so): Should we also make this configurable?

Reply:
I would suggest we do this when someone requests it. The configuration becomes more and more complex, and I would like to avoid too many changes for now.

Reply:
Sounds reasonable to me.